src/GovUk.Education.ExploreEducationStatistics.Admin/Services/DataGuidanceService.cs

```diff
@@ -132,6 +133,15 @@ public class DataGuidanceService : IDataGuidanceService
             {
                 var content = updateRequestsByFileId[releaseFile.FileId].Content;
 
+                _contentDbContext.Update(releaseFile);
+                releaseFile.Summary = content;
```

Comment:
The `Summary` column was already on `ReleaseFile`? So I guess it wasn't being used?

Reply:
Yep that's correct, it's not used for ReleaseFiles where FileType is 'Data'.

We've had some discussions recently about this not being ideal due to it mixing html and non-html text in the `Summary` field. `Summary` is already being used for other types of files like ancillary where it's not html.

Ideally we'd refactor files of type 'Data' and 'Metadata' to be a single DataSet entity, and then we might have something like a DataSetVersion or a ReleaseDataSet where the guidance would belong.

We said that this refactoring would be out of scope for this ticket.

There's also a plan to strip html from the data guidance in the near future and tidy up any guidance for data sets that's only repeating what's already found in the metadata, so that the guidance is more like a summary. We need cleaner guidance to be displayed as summaries in the data catalogue.